engine/src/test/java/com/datatorrent/stram/plan/physical/PhysicalPlanTest.java

```diff
@@ -20,25 +20,43 @@
 
 
 import java.io.Serializable;
-import java.util.*;
```

Comment:
Please remove the import changes. That was already done in master and we want only the net diff for this patch.